feat(banner): Using picture instead of the background image - FRONT-3872

planctus opened the pull request.

emeryro commented on src/implementations/vanilla/components/banner/_banner.scss:
```diff
@@ -107,28 +112,28 @@ $_font-weight: null !default;
+    }
+  }
+
+  .ecl-banner--plain-background::before {
```
do we still need this? I think that it was for the ratio

planctus replied:
But nothing has changed regarding the plain-background variant, there is no image so the aspect-ratio is still set by this css hack.

emeryro commented on src/implementations/vanilla/components/banner/_banner.scss:
```diff
@@ -184,18 +189,12 @@ $_font-weight: null !default;
 
+  // manage ratio 4:1
   .ecl-banner__image {
+    aspect-ratio: 4/1;
```
this aspect ratio overrides the one from the banner size, so changing size does not work anymore

emeryro reviewed (changes requested):
Image is not cropped the same way as before (both desktop and mobile)
![image](https://user-images.githubusercontent.com/25579221/229438900-9cb81bee-1134-41b3-9930-7ade6ba78008.png)
![image](https://user-images.githubusercontent.com/25579221/229438928-7dd4fe4e-42fb-4e68-b6f3-da6d229c0e69.png)

planctus commented:
@emeryro , we replaced a background image with an img tag, this change in the way the image is rendered is "intended", now we use object-fit, before we had a background-size, they do not behave the same.
I've added a object-position to reproduce the previous behaviour, but i'm not really confident this is what we should do for every image.
